src/remarkable/remarkable.tsx

```diff
@@ -15,10 +18,24 @@ md.renderer = new RemarkableReactRenderer({
         code: code,
         pre: pre,
         table: table,
-        img: img
+        img: img,
+        video: video,
+        attachment: attachment
+    },
+    tokens: {
+        link_video_open: 'video',
+        link_attachment_open: 'attachment'
     }
 });
 
+const rules = md.inline.ruler.getRules('');
+const origParser = rules.find((rule) => rule.name === 'links');
```

Comment:
Brackets are not necessary around "rule"

Reply:
Prettier adds them automatically.